.eslintrc.js

```diff
@@ -43,6 +43,10 @@ module.exports = {
       }
     }],
     'one-var': ['error', 'never'],
+    'import/no-extraneous-dependencies': ['error', {
+      /* the following folders can import devDependencies */
+      devDependencies: ['test/**', 'tests/**', 'src/debug/**', 'examples/**'],
```

Comment:
Btw our npm stuffs are broken too, if I'm not mistaken:

- we ignore src/
- we don't include any `dist/` folder even though we reference it in the main of our package.json.

Shouldn't we include src, and include dist too? (and update dist each time we release?)